### PR TITLE
fix: modified the json-schema for linz properties to copy stac

### PR DESCRIPTION
**Pull request by MitchellPaff**

### CODE REVIEW

File: extensions/linz/schema.json [hunk 1]
```diff
@@ -16,14 +16,16 @@
           "minLength": 1
         },
         "created": {
+          "title": "Creation Time",
           "type": "string",
-          "minLength": 1,
-          "format": "date-time"
+          "format": "date-time",
+          "pattern": "(\\+00:00|Z)$"
```
Comment (l0b0) [inline extensions/linz/schema.json:22]: Why require a time zone specifier? [The standard already specifies that the datetime needs to be UTC](https://github.com/radiantearth/stac-spec/blob/f81324f712a608d4a319fd0a8738193a43ef1d51/item-spec/common-metadata.md#date-and-time).

File: extensions/linz/README.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ extension which adds constraints to default STAC schema properties.
...
-| updated                 | string | **REQUIRED**. Last updated Datetime.                                                                                                                                                                                                                                                                                                             |
+| created                 | string | **REQUIRED**. Creation Time.                                                                                                                                                                                                                                                                                                                     |
+| updated                 | string | **REQUIRED**. Last Update Time.                                                                                                                                                                                                                                                                                                                  |
```
Comment (l0b0) [inline extensions/linz/README.md:30]: Could we copy the [descriptions from the standard](https://github.com/radiantearth/stac-spec/blob/f81324f712a608d4a319fd0a8738193a43ef1d51/item-spec/common-metadata.md#date-and-time), or reference that section of the standard, for clarity?

File: extensions/linz/schema.json (hunk 1 above)
Comment (MitchellPaff) [reply extensions/linz/schema.json:22]: https://github.com/linz/geostore/issues/978#issuecomment-923451562

Comment (MitchellPaff) [reply extensions/linz/schema.json]: https://linz.slack.com/archives/CN7F6HP2Q/p1632178810030700

don't think so. These are new fields meant for the collection metadata- those dates eist in the parent schema only for Items i beilieve.

**Review by l0b0 — APPROVED** [review]

**Pull request merged**